R/board_rsconnect.R

```diff
@@ -35,15 +35,15 @@
 #'
 #' @inheritParams new_board
 #' @inheritParams board_url
-#' @param auth There are two approaches to auth: you can either use `"envvars"`
-#'   `CONNECT_API_KEY` and `CONNECT_SERVER` or the rsconnect package. The
-#'   default is `auto`, which will use the environment variables if both are
-#'   available, and rsconnect if not.
-#' @param server For `auth = "envvar"` the full url to the server, like
-#'   `http://server.rstudio.com/rsc` or `https://connect.rstudio.com/`.
+#' @param auth There are three approaches to auth: use `"manual"` and provide `server` and `key`,
+#'   use `"envvars"`  `CONNECT_API_KEY` and `CONNECT_SERVER`, or the rsconnect package. The
```

Comment:
```suggestion
#'   use `"envvars"`  `CONNECT_API_KEY` and `CONNECT_SERVER`, or use the "rsconnect" package. The
```

I wonder if it might be clearer to switch this to a bullet list?